.github/workflows/build.yml

```diff
@@ -32,8 +32,15 @@ jobs:
       - name: Build Main
         run: |
           mvn -Dorg.apache.activemq.artemis.core.io.aio.AIOSequentialFileFactory.DISABLED=AnythingNotNull -Djdk8-errorprone -Pfast-tests -Pextra-tests -Ptests-CI -Pjmh install
-      - name: Build Examples
+
+      - name: Build Examples (JDK8 / -Prelease)
+        if: matrix.java == '8'
```

Comment:
@gemmellr before last release, I had to fix javadoc, however currently javadoc only builds on JDK 1.8

I tried to do an if jdk=8 at the time but I did not know how to do it?

Since you figured it out now, can we add after this line?

```mvn -Prelease javadoc:javadoc```


WDYT?

Reply:
I wanted to prevent people breaking javadocs again.. or at least be aware when that happens.